specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -2266,6 +2474,123 @@
       ],
       "description": "The List hybrid machine operation response."
     },
+    "LicenseProfile": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/LicenseProfileProperties",
```

Comment:
I think we should wrap all the properties we're currently adding in one more layer, so that we can use licenseProfile for more things going forward without getting it intermingled with the ArcESU properties. what I mean is, lets model it as:
```
{
  ...
  "properties": {
    "esuProfile": {
      <all the properties>
    }
  }
}
```